.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,50 @@
+name: Publish Documentation Site
+
+on:
+  push:
+    branches: [stromberg/manual-publish]
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch'
+        default: 'main'
+        required: true
+
+jobs:
+  build-docs:
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout Branch
+      uses: actions/checkout@v2.3.3
+      with:
+        ref: ${{ github.event.inputs.name }}
+        path: ${{ github.event.inputs.name }}
+
+    # Ruby Gems
+    - name: Load gem cache
+      uses: actions/cache@v2.1.2
+      with:
+        path: ${{ github.event.inputs.name }}/.bundle
+        key: gems-${{ hashFiles('Gemfile.lock') }}
+
+    - name: Set up Swift environment
+      run: |
+        # Set global bundle path so it gets used by build_swift_docs.sh running in the nested repo as well.
+        cd "${{ github.event.inputs.name }}"
+        bundle config --global path "$(pwd)/.bundle"
+        bundle check || bundle install
+        # Don't need to run pod gen, the website script does that itself.
+        brew install sourcedocs
+        sudo xcode-select -s /Applications/Xcode_11.4.app
```

Comment:
Isn't this a pretty old version of Xcode?

Reply:
Ish. Right now, our language version targets Swift 5.2, which was released with Xcode 11.4. We could probably bump this up, but I'm fine leaving this as-is for now.